hal/network/lwip/cellular/pppncpnetif.cpp

```diff
@@ -102,6 +102,7 @@ void PppNcpNetif::loop(void* arg) {
         self->celMan_->ncpClient()->enable(); // Make sure the client is enabled
         if (!r) {
             // Event
+            LOG(TRACE, "PPP netif event from queue: %d", ev);
```

Comment:
TODO: peek events off of the queue and retry in case of failure.

Reply:
I'd like to achieve this feature in a separated PR, sounds good?